client/client.go

```diff
@@ -281,21 +281,93 @@ func newHttpClient() *http.Client {
 	return &http.Client{}
 }
 
+func loadServerTrust(ctx *openssl.Ctx, conf *Config) (*openssl.Ctx, error) {
+	// Trust CA's in the standard location (/etc/ssl/certs), and the
+	// configured server certificate when building the certificate chain
+	err := ctx.LoadVerifyLocations(conf.ServerCert, "/etc/ssl/certs/")
```

Comment:
For now, hardcoded `/etc/ssl/certs`, but this should be changed before merging, when the std-certpool loading from @merlin-northern  is in place.

Reply:
thats right. I would like to get this one first, then [pull/587](https://github.com/mendersoftware/mender/pull/587) and then the follow up patch here removing the hardcoded path.

Reply:
Question:
Why not do it the other way around? First #587 and then this?

Reply:
That means we can clean this up also in this same PR: https://github.com/mendersoftware/mender/pull/588/files#r471329025

Reply:
> Question:
> Why not do it the other way around? First #587 and then this?

the only reason: I figured it would be easier for me to apply #587 on top of this one, than for you to resolve the conflicts. but if you are fine with getting #587 first, then I am all for it!

Reply:
Let's do it that way then. I don't mind resolving them, and it would be nice to get this PR and commit clean with most all the `OpenSSL` stuff contained in one nice and clean PR, considering it is a pretty solid functionality shift.

Reply:
ok, https://github.com/mendersoftware/mender/pull/587 has been merged (with the required meta-mender PR)

Reply:
Rebased now. Looks good.